docs/generate_metadata.py

```diff
@@ -0,0 +1,40 @@
+import pathlib
+import datetime
+
+import atlalign
+
+metadata_template = \
```

Comment:
Maybe for a slightly better look one could replace
```python
metadata_template = """---
packageurl: ...
"""
```
by
```python
metadata_template = """\
---
packageurl: ...
"""
```
it should produce the same string.